common/openapi.yml

```diff
@@ -145,6 +145,25 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/v1/datasets/{id}/export:
```

Comment:
Thoughts about this endpoint?

Reply:
I think `export` makes sense. (vs `link` which we aren't planning to do at this point).